packages/manager/apps/catalog/e2e/utils/network.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ICustomWorld,
+  toPlaywrightMockHandler,
+  Handler,
+} from '../../../../../../playwright-helpers';
```

Comment:
Same as previous

Reply:
No it doesn't work
The alias in tsconfig.json is not taken into account